Makefile

```diff
@@ -28,6 +28,8 @@ help:
 	@echo "  reindex           to reindex everything in elasticsearch, for AMO"
 	@echo "  flake8            to run the flake8 linter"
 	@echo "Check the Makefile  to know exactly what each target is doing. If you see a "
+	@echo "target using something like $(SETTINGS), you can make it use another value:"
+	@echo "  make SETTINGS=settings_mine docs"
```

Comment:
That doesn't make it not wrong :p

Sent fron phome. Please ecxuse the fact that I can't tyoe.
On Aug 26, 2015 3:03 PM, "wagnerand" <notifications@github.com> wrote:

> In Makefile
> <https://github.com/mozilla/olympia/pull/691#discussion_r38040775>:
>
> > @@ -28,6 +28,8 @@ help:
> >  	@echo "  reindex           to reindex everything in elasticsearch, for AMO"
> >  	@echo "  flake8            to run the flake8 linter"
> >  	@echo "Check the Makefile  to know exactly what each target is doing. If you see a "
> > +	@echo "target using something like $(SETTINGS), you can make it use another value:"
> > +	@echo "  make SETTINGS=settings_mine docs"
>
> This is exactly as it used be before it got accidentally deleted.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/olympia/pull/691/files#r38040775>.
>